lib/utils/token.js

```diff
@@ -0,0 +1,11 @@
+var jwt = require('jsonwebtoken');
```

Comment:
```suggestion
require('dotenv').config();
var jwt = require('jsonwebtoken');
```
The dotenv module should be required on the first line of the file it's used in.